### ISSUE TITLE
Loss patterns in toy data for lora training.

### ISSUE THREAD

**Issue by LearningLeopard** [VectorSpaceLab/OmniGen#208]
Hey OmniGen team,
Thank you for such easy to use fine tuning scripts. It was very easy to get started. I was playing around with your toy data and when I'm training the model with that, the loss patterns are fluctuating in a very unstable way. Now I know from general experience that it could mean that lr is high. Now even when I tried to reduce it, the loss history has not stabilized. Now I'm new to training T2I models and specially with lora, is this fluctuation normal or not. Basically, I am going to train with some of my own data and if doing so if I find again such similar loss fluctuation patterns, is it okay or should I play with the hyper parameters for more stable pattern?

Thank you for your inputs!

**Comment by Santiagoelao** [comment]
I believe I found a solution to your issue. Check it out here [click](https://forreststonesolutions.com/robots/)

**Comment by Yukino912** [comment]
@LearningLeopard I encountered the same problem. May I ask if you have solved it

**Comment by LearningLeopard** [comment]
Hey @Yukino912 , I used the fine-tuned model trained on my data anyway and it did improve as expected(i.e., I've seen a similar improvement in other models in par with this).

**Comment by ayazmhmd** [comment]
I encouter the same issue while fine tuning on my custom data the loss remains the same
